Source/PeleLMPlot.cpp

```diff
@@ -53,7 +53,7 @@ void PeleLM::WritePlotFile() {
    const std::string& plotfilename = amrex::Concatenate(m_plot_file, m_nstep, m_ioDigits);
 
    if (m_verbose) {
-      amrex::Print() << " Dumping plotfile: " << plotfilename << "\n";
+      amrex::Print() << "\n Writting plotfile: " << plotfilename << "\n";
```

Comment:
Typo on writing

Reply:
Ah ah ! Indeed, not much cleaner than `Dumping` ...